tests/src/dir.targets

```diff
@@ -66,7 +66,6 @@
   build a run only for a higher priority testcase.
   -->
   <Target Name="Build" Condition="('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)') Or ('$(_CLRTestInServiceOfHigherPriority)')">
-    <MSBuild Projects="@(ProjectReference)" Properties="_CLRTestInServiceOfHigherPriority=true"/>
```

Comment:
You're correct about the priority system for filtering the projects. I like your 2nd suggestion considering that no tests are currently 'In service of higher priority'. I'll give it a whirl.